lib/THNN/generic/BatchNormalization.c

```diff
@@ -10,10 +10,10 @@ void THNN_(BatchNormalization_updateOutput)(
   bool train, double momentum, double eps)
 {
   long nInput = THTensor_(size)(input, 1);
-  long n = THTensor_(nElement)(input) / nInput;
+  long f,n = THTensor_(nElement)(input) / nInput;
 
   #pragma omp parallel for
-  for (long f = 0; f < nInput; ++f) {
+  for (f = 0; f < nInput; ++f) {
```

Comment:
if you make this change, then you'll have to change the pragma to: #pragma omp parallel for private(f)